samples/local.settings.json

```diff
@@ -3,6 +3,7 @@
   "Values": {
     "AzureWebJobsStorage": "UseDevelopmentStorage=true",
     "FUNCTIONS_WORKER_RUNTIME": "dotnet",
-    "SqlConnectionString": ""
-}
+    "SqlConnectionString": "",
+    "AzureFunctionsSqlBindingsTelemetryOptOut": true
```

Comment:
Are we setting this to true to avoid our tests from inflating numbers?

Reply:
Correct

Reply:
Moved this to just be set by the env var when running tests